app-k9mail/build.gradle.kts

```diff
@@ -152,3 +157,44 @@ android {
 dependencyGuard {
     configuration("releaseRuntimeClasspath")
 }
+
+tasks.create("printVersionInfo") {
```

Comment:
I think, as next step we could move this to the `build-plugin` and generalize this for both apps. As reference `thunderbird.quality.badging.gradle.kts`